web/happenings/templates/happenings/my_events_list_view.html

```diff
@@ -4,49 +4,92 @@
-                                  <div class="description">
-                                    <p>{{ event.description }}</p>
-                                  </div>
+                                    <div class="header">{{ event.name }}</div>
+                                    <div class="description">
+                                        <p>{{ event.description }}</p>
+                                    </div>
                                 </div>
                             </a>
                         {% endfor %}
+
+
                     </div>
                 {% else %}
-                    <p>You haven't added any events yet!</p>
+                    <p>You don't have any upcoming/current!</p>
                 {% endif %}
+            </div>
 
+            
+            <div id="old_events" class="content" > 
+                    <div class="ui huge header"> Old-events </div>
+                    {% if my_events_list.expired %}
+                        <div class="ui link items">
+                            {% for event in my_events_list.expired %}
+                                <a class="item" href="{% url 'my_events_detailed' event.id %}">
+                                    <div class="content">
+                                        <div class="header">{{ event.name }}</div>
+                                        <div class="description">
+                                            <p>{{ event.description }}</p>
+                                        </div>
+                                    </div>
+                                </a>
+                            {% endfor %}
+                        </div>
+                    {% else %}
+                        <p>You don't have any old events!</p>
+                    {% endif %}
+                    
             </div>
+
+            <script language="javascript" type="text/javascript">
+                document.getElementById("old_events").style.display = "none";
+            </script>
+
             <div class="extra content">
-     
                 <a class="ui green button" href="{% url 'create_event' %}">Create Event</a>
-  
+                <button class="ui brown button" onclick="hide_div('old_events')">Old events</button>
             </div>
-
         </div>
     </div>
+    
     {#For your swipes#}
     <div class="ui container">
         <div class="ui card">
             <div class="content">
-                <div class="header">Your swipes!</div>
-                    <br>
-                    <div class="meta">You haven't swiped any events yet!</div>
+                <div class="ui huge header">Your swipes!</div>
+                    {% if my_events_list.my_swipes %}
+                        <div class="ui divided link items">
+                            {% for schedule in my_events_list.my_swipes %}
+                                <a class="item" href="{% url 'my_events_detailed' schedule.event.id %}">
```

Comment:
I'm not sure if "my swipes" should send you to "my_events_detailed". For me it would make sense for the link to go to "events" (The same site as events shown when you go through the event list).